content/en/docs/OIDC/users_and_roles.md

```diff
@@ -42,6 +42,15 @@ To create a user, refer to the following documentation:
 
 ## Configure Roles for Each Tenant in Cozy:
 
+### Cluster wide
```

Comment:
As we are adding external integrations now, in the "Keycloak Address" section above, I'd add a note:

"If  you are planning to integrate with external services either as clients or as IdPs, your Keycloak address needs to be publicly accessible and reachable by these services."